VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/select-hearing-layout.component.spec.ts

```diff
@@ -72,7 +72,7 @@ describe('SelectHearingLayoutComponent', () => {
 
     it('should return recommended layout', () => {
         component.selectedLayout = undefined;
-        expect(component.getSelectedOrPreferredLayout()).toBe(HearingLayout.Dynamic);
+        expect(component.getSelectedOrPreferredLayout()).toBe(HearingLayout.OnePlus7);
```

Comment:
shout this have changed?

Reply:
It had too as the addition of the extra person changed the suggested Layout

Reply:
if that's the case could you ensure we still have tests to cover recommend layouts for less than 5 participants?